packages/utilities/psammead-locales/src/numerals/index.js

```diff
@@ -0,0 +1,97 @@
+export const Bengali = ['০', '১', '২', '৩', '৪', '৫', '৬', '৭', '৮', '৯', '১০'];
```

Comment:
Curious why this package exports 0-10 rather than just 0-9?

Reply:
We're going to use these for Most Read and it's convenient to have the value for `10` available rather than have the component that uses this to construct the number `10`.